### PR TITLE
CLK-1631 introducing rubocop and bundler-audit

### PR DESCRIPTION
**Pull request by stefan-hc**
This is copy-paste of Clockwork configuration for rubocop.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,258 @@
+require:
```
Comment (stefan-hc) [inline .rubocop.yml:1]: This file was copied from Clockwork.

File: Gemfile [hunk 2]
```diff
@@ -2,5 +2,15 @@ source 'https://rubygems.org'
...
 
+def gems(*gems)
+  gems.each { |g| gem(g) }
+end
+
 gem 'addressable'
 gem 'faraday', '~> 0.11.0'
+
+group :development do
+  gems "bundler-audit",
```
Comment (stefan-hc) [inline Gemfile:13]: "bundler-audit" is not needed for Rubocop, but I'm adding it anyway. In one of the next steps we will/might add it to CI as separate step.

File: Gemfile [hunk 3]
```diff
@@ -2,5 +2,15 @@ source 'https://rubygems.org'
 
 gemspec
 
+def gems(*gems)
```
Comment (skv-headless) [inline Gemfile:5]: Just curious what is it for?

**Review by skv-headless — APPROVED** [review]

File: Gemfile (hunk 3 above)
Comment (stefan-hc) [reply Gemfile:5]: It's some copy-paste from Gemfile of Clockwork. I guess this is some Paweł's shortcut for omitting writing `gem` in each line... :shrug:

Comment (pawelniewie) [reply Gemfile:5]: I would actually remove it. I don't like `gems` anymore.

**Review by pawelniewie — APPROVED** [review]

**Pull request merged**